tf/modules/postgresql/variables.tf

```diff
@@ -48,7 +48,7 @@ variable "db_max_allocated_storage" {
 }
 
 variable "db_engine_version" {
-  default = "16.3"
+  default = "16.4"
```

Comment:
I changed this in here since we had done an upgrade of postgresql directly from the web interface of AWS